specification/redis/resource-manager/Microsoft.Cache/preview/2023-05-01/redis.json

```diff
@@ -2841,6 +3312,15 @@
       "required": true,
       "type": "string",
       "description": "Client Api Version."
+    },
```

Comment:
Are these still referenced anywhere? Maybe we can remove them.

Reply:
Removed